Fix offline installation page with offline-specific setup wizard instructions (#9180)
jenkins-infra/jenkins.io: content/doc/book/installing/offline.adoc

assemble plugins.

[[setup-wizard]]
== Post-installation setup wizard

After installing Jenkins offline, the post-installation setup wizard begins.

=== Unlocking Jenkins

When you first access a new Jenkins controller, you are asked to unlock it using an automatically-generated password.

. Browse to `\http://localhost:8080` and wait until the *Unlock Jenkins* page appears.
. From the Jenkins console log output, copy the automatically generated alphanumeric password (between the 2 sets of asterisks).
. On the *Unlock Jenkins* page, paste this password into the *Administrator password* field and click *Continue*.

NOTE: The initial admin password can be found at `/var/lib/jenkins/secrets/initialAdminPassword`. Run `sudo cat /var/lib/jenkins/secrets/initialAdminPassword` to print it in the console.

=== Customizing Jenkins with plugins

After unlocking Jenkins, the *Customize Jenkins* page appears.

NOTE: Because this is an offline installation, Jenkins cannot download plugins. To skip plugin installation during setup:

. Click *Select plugins to install*.
. Click *None* to deselect all suggested plugins.
. Click *Install* to proceed without downloading plugins.

> You can install plugins manually later using the link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin Installation Manager Tool] described above.

Reviewer: I think it is better to link to the section of the document that describes alternatives rather than linking to the plugin installation manager tool.
```suggestion
You can use one of the techniques described in link:#offline-plugin-installation[offline plugin installation] to install plugins.
```

Author: hello @MarkEWaite, thank you for the suggestion!
I've updated the link to point to the offline 
plugin installation section instead.

Jenkins and plugins through Nexus or Artifactory since Gradle is used to
assemble plugins.

[[setup-wizard]]
== Post-installation setup wizard

After installing Jenkins offline, the post-installation setup wizard begins.

=== Unlocking Jenkins

When you first access a new Jenkins controller, you are asked to unlock it using
an automatically-generated password.

. Browse to `\http://localhost:8080` and wait until the *Unlock Jenkins* page appears.
. From the Jenkins console log output, copy the automatically generated

>   alphanumeric password (between the 2 sets of asterisks).

Reviewer 2: ```suggestion
. From the Jenkins console log output, copy the automatically generated alphanumeric password (between the 2 sets of asterisks).
```

Reviewer 2: We prefer one line per sentence as per the style guide.

. On the *Unlock Jenkins* page, paste this password into the *Administrator

>   password* field and click *Continue*.

Reviewer 2: ```suggestion
. On the *Unlock Jenkins* page, paste this password into the *Administrator password* field and click *Continue*.
```

Reviewer 2: And can you tell us more about how to get this admin password?

Author: Hello @krisstern, thank you for the review!

I have made the following changes as requested:
- Fixed formatting to one line per sentence as per style guide
- Added NOTE explaining how to retrieve the initial admin password using:
  `sudo cat /var/lib/jenkins/secrets/initialAdminPassword`
- Fixed link formatting to one line

=== Customizing Jenkins with plugins

After unlocking Jenkins, the *Customize Jenkins* page appears.

NOTE: Because this is an offline installation, Jenkins cannot download plugins.
To skip plugin installation during setup:

. Click *Select plugins to install*.
. Click *None* to deselect all suggested plugins.
. Click *Install* to proceed without downloading plugins.

You can install plugins manually later using the
link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin Installation Manager Tool]

> described above.

Reviewer 2: ```suggestion
link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin Installation Manager Tool] described above.
```
